compose/project.py

```diff
@@ -13,6 +14,8 @@
 
 log = logging.getLogger(__name__)
 
+VolumeFrom = namedtuple("VolumeFrom", 'source mount_option')
```

Comment:
For the sake of consistency, could you:

1. rename `mount_option` to `mode`
2. rename `VolumeFrom` to `VolumesFromSpec`
3. move it into `compose.service` and import it here